src/majordomo/include/majordomo/Broker.hpp

```diff
@@ -65,14 +66,14 @@ struct fmt::formatter<opencmw::majordomo::detail::DnsServiceItem> {
     template<typename FormatContext>
     auto format(const opencmw::majordomo::detail::DnsServiceItem &v, FormatContext &ctx) const {
 #if not defined(__EMSCRIPTEN__) and (not defined(__clang__) or (__clang_major__ >= 16))
-        return fmt::format_to(ctx.out(), "[{}: {}]", v.serviceName, fmt::join(v.uris | std::views::transform(opencmw::majordomo::detail::uriAsString), ","));
+        return std::format_to(ctx.out(), "[{}: {}]", v.serviceName, opencmw::join(v.uris | std::views::transform(opencmw::majordomo::detail::uriAsString), ","));
 #else
         std::vector<std::string> uris{};
         uris.reserve(v.uris.size());
         for (auto &uri : v.uris) {
             uris.emplace_back(opencmw::majordomo::detail::uriAsString(uri));
         }
-        return fmt::format_to(ctx.out(), "[{}: {}]", v.serviceName, fmt::join(uris, ","));
+        return std::format_to(ctx.out(), "[{}: {}]", v.serviceName, std::join(uris, ","));
```

Comment:
```suggestion
        return std::format_to(ctx.out(), "[{}: {}]", v.serviceName, opencmw::join(uris, ","));
```
possible oversight, but this would mean that this branch of the file is not compiled at all? probably because broker is only compiled for GCC?